style.css

```diff
@@ -0,0 +1,84 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 24px;
+  margin-right: 16px;
+  margin-top: 58px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.logo a {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  width: 375px;
+  height: 718px;
+  background-image: url("./Images/header-shapes mobile.png");
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 70px 24px;
+}
```

Comment:
- [ ] Kindly make sure your `background image` covers the entire page to match the Figma design and to make your pages look smart. I suggest you change the `width: 375px;` to `width: 100vw;` and add add the `background-size: 100% or cover;`. After that, you can use the correct given properties by Figma design to match, such as: `size`, `margin`, `paddings`, `fonts`,`etc.`
- Figma design:
![Figma_1](https://user-images.githubusercontent.com/83042742/170516660-b0867e34-707b-4302-8396-eeecd9740cb4.png)
- Your design:
![now](https://user-images.githubusercontent.com/83042742/170516855-53a0ffaf-a373-475f-9ac0-a5fde9947acb.png)